java-wrapper/src/main/java/com/codedisaster/steamworks/SteamMatchmaking.java

```diff
@@ -487,7 +487,7 @@ private static native boolean getLobbyDataByIndex(long pointer, long steamIDLobb
 		ISteamMatchmaking* matchmaking = (ISteamMatchmaking*) pointer;
 		MatchMakingKeyValuePair_t result;
 		bool success = matchmaking->GetLobbyDataByIndex((uint64) steamIDLobby, lobbyDataIndex,
-			result.m_szKey, 256, result.m_szValue, 256);
+			result.m_szKey, 256, result.m_szValue, 8192);
```

Comment:
What's the reason behind this? The native field has a fixed size of `char[256]`.

Reply:
According to the Steamworks Documentation (https://partner.steamgames.com/doc/api/ISteamMatchmaking) the cchValueBufferSize should be set to "k_cubChatMetadataMax" which is the "Maximum size in bytes that chat room, lobby, or chat/lobby member metadata may have.". It's value is "8192", so I thought it should be called with 8192 instead (maybe I misinterpreted this paragraph?).

Reply:
Ah, I see. Ok, then the bug is that you shouldn't use MatchMakingKeyValuePair_t for storing the result, because its m_szValue field is only limited to 256 characters.